TimeTracker/Tracker/Tracker.cs

```diff
@@ -8,16 +8,20 @@ namespace nsTracker
 {
     public class Tracker
     {
-        public Tracker(List<Task>tasks)
+        public Tracker(List<Tasks>tasks)
         {
             this._tasks = tasks;
             this._database = new CSV();
         }
 
-        public void Update()
+        public void Update(int selection)
```

Comment:
Do you still need this method? Why don't you use directly Update Pressed Button ?